SlevomatCodingStandard/Sniffs/Classes/RequireSingleLineMethodSignatureSniff.php

```diff
@@ -60,4 +73,26 @@ public function process(File $phpcsFile, $methodPointer): void
 		$phpcsFile->fixer->endChangeset();
 	}
 
+	/**
+	 * @param string $methodSignature
+	 * @param string[] $patterns
+	 * @return bool
+	 */
+	private function isMethodSignatureInPatterns(string $methodSignature, array $patterns): bool
+	{
+		foreach (SniffSettingsHelper::normalizeArray($patterns) as $pattern) {
```

Comment:
Please use `normalizedPatters` so the method `SniffSettingsHelper::normalizeArray()` is not called everytime.

See Eg. https://github.com/slevomat/coding-standard/blob/master/SlevomatCodingStandard/Sniffs/Commenting/UselessFunctionDocCommentSniff.php#L130